users/views/user_invitation_view.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render
+from django.contrib import messages
+from django.contrib.auth.models import User
+from users.forms import UserInvitationForm
+from django.views.generic import View
+from users.models import SignupInvitation
+from django.core.mail import send_mail
+
+
+class UserInvitationView(View):
+    template_name = "registration/send_invitation.html"
+
+    def get(self, request):
+        form = UserInvitationForm()
+        return render(request, self.template_name, {'form': form})
```

Comment:
The invitation list must appear on this method too

Reply:
invitations = SignupInvitation.objects.filter(invited_by=request.user)
return render(request, self.template_name, {'invitations': invitations, 'form': form})

Reply:
Done.